### ISSUE TITLE
Error scoring on Marques2020_Ringach2002, AssertionError

### ISSUE THREAD

**Issue by YudiXie** [brain-score/vision#656]
I am getting this error when I see the scoring process log for the model I submitted. I am wondering if this happens because this benchmark requires some specific model definition? 

```
Traceback (most recent call last):
  File "/scratch2/weka/quest/shared/jenkins/miniconda3/envs/yudixie_resnet50_imagenet1kpret_0_240312_Marques2020_Ringach2002-or_bandwidth/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/scratch2/weka/quest/shared/jenkins/miniconda3/envs/yudixie_resnet50_imagenet1kpret_0_240312_Marques2020_Ringach2002-or_bandwidth/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "brainscore_vision/__main__.py", line 20, in <module>
    fire.Fire()
  File "/scratch2/weka/quest/shared/jenkins/miniconda3/envs/yudixie_resnet50_imagenet1kpret_0_240312_Marques2020_Ringach2002-or_bandwidth/lib/python3.7/site-packages/fire/core.py", line 143, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "/scratch2/weka/quest/shared/jenkins/miniconda3/envs/yudixie_resnet50_imagenet1kpret_0_240312_Marques2020_Ringach2002-or_bandwidth/lib/python3.7/site-packages/fire/core.py", line 482, in _Fire
    target=component.__name__)
  File "/scratch2/weka/quest/shared/jenkins/miniconda3/envs/yudixie_resnet50_imagenet1kpret_0_240312_Marques2020_Ringach2002-or_bandwidth/lib/python3.7/site-packages/fire/core.py", line 693, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "brainscore_vision/__main__.py", line 9, in score
    result = _score_function(model_identifier, benchmark_identifier, conda_active)
  File "/rdma/vast-rdma/scratch/Sun/score_plugins_vision_env_163/vision/brainscore_vision/__init__.py", line 105, in score
    score_function=_run_score, conda_active=conda_active)
  File "/scratch2/weka/quest/shared/jenkins/miniconda3/envs/yudixie_resnet50_imagenet1kpret_0_240312_Marques2020_Ringach2002-or_bandwidth/lib/python3.7/site-packages/brainscore_core/plugin_management/conda_score.py", line 88, in wrap_score
    result = score_function(model_identifier, benchmark_identifier)
  File "/rdma/vast-rdma/scratch/Sun/score_plugins_vision_env_163/vision/brainscore_vision/__init__.py", line 76, in _run_score
    score: Score = benchmark(model)
  File "/rdma/vast-rdma/scratch/Sun/score_plugins_vision_env_163/vision/brainscore_vision/benchmark_helpers/properties_common.py", line 46, in __call__
    pos=stim_pos)
  File "/rdma/vast-rdma/scratch/Sun/score_plugins_vision_env_163/vision/brainscore_vision/benchmark_helpers/properties_common.py", line 100, in filter_receptive_fields
    rf_pos, rf_map = map_receptive_field_locations(model_identifier=model_identifier, model=model, region=region)
  File "/scratch2/weka/quest/shared/jenkins/miniconda3/envs/yudixie_resnet50_imagenet1kpret_0_240312_Marques2020_Ringach2002-or_bandwidth/lib/python3.7/site-packages/result_caching/__init__.py", line 80, in wrapper
    result = function(*args, **kwargs)
  File "/rdma/vast-rdma/scratch/Sun/score_plugins_vision_env_163/vision/brainscore_vision/benchmark_helpers/properties_common.py", line 115, in map_receptive_field_locations
    _assert_grating_activations(rf_activations)
  File "/rdma/vast-rdma/scratch/Sun/score_plugins_vision_env_163/vision/brainscore_vision/benchmark_helpers/properties_common.py", line 232, in _assert_grating_activations
    == nStim
AssertionError
```

**Comment by samwinebrake** [comment]
Looking into this now. I'll let you know what I find!

**Comment by mike-ferguson** [comment]
Addressed in #711
